### ISSUE TITLE
Server status check dashboard

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-project#51]
Currently there are more ways to check the status of the server:
- REST endpoints are monitored by outside service
- Back end services can be also monitored from server shell

It would be useful to have an "integrated view" of all the components.
This would be a REST call which actually tries to access:
- All the other REST enpoints, or at least one on each server
- All the back end services - doing this by opening a connection to the port that should listed.
- Or by doing a dedicated call by the native technology:
  - execute a Mongo query
  - execute a Cypher query
  - do a REST call to Keycloak

**Comment by graybeal** [comment]
This should leverage expertise IT group is developing in automatic status test/reporting tools.

Sent from my iPhone

On May 24, 2016, at 5:53 AM, martinjoconnor <notifications@github.com<mailto:notifications@github.com>> wrote:


Currently there are more ways to check the status of the server:

  *   REST endpoints are monitored by outside service
  *   Back end services can be also monitored from server shell

It would be useful to have an "integrated view" of all the components.
This would be a REST call which actually tries to access:

  *   All the other REST enpoints, or at least one on each server
  *   All the back end services - doing this by opening a connection to the port that should listed.
  *   Or by doing a dedicated call by the native technology:
     *   execute a Mongo query
     *   execute a Cypher query
     *   do a REST call to Keycloak

**Comment by marcosmro** [comment]
Here is a recent experience that supports the need for the status check dashboard:
Recently, the Redis (persistent) service was not correctly started in our production machine. We noticed that something was wrong when we tried to create a template from the UI and the 'green box' was not shown. The POST call was returning a timeout but the template was created correctly. We need a way of checking the status of all server components to ensure that the system is always 100% operational.
